hazelcast/src/main/java/com/hazelcast/map/impl/operation/MultipleEntryOperation.java

```diff
@@ -104,7 +104,12 @@ public int getAsyncBackupCount() {
     @Override
     public Operation getBackupOperation() {
         EntryBackupProcessor backupProcessor = entryProcessor.getBackupProcessor();
-        return backupProcessor != null ? new MultipleEntryBackupOperation(name, keys, backupProcessor) : null;
+        MultipleEntryBackupOperation backupOperation = null;
+        if (backupProcessor != null) {
+            backupOperation = new MultipleEntryBackupOperation(name, keys, backupProcessor);
+            backupOperation.setWanEventList(wanEventList);
```

Comment:
Did I get it right that this `setWanEventList()` call copies the existing events from the `MultipleEntryOperation` instead of creating new ones in the `AbstractMultipleEntryBackupOperation` via `wanEventList.add(new WanEventWrapper(key, null, EntryEventType.REMOVED|UPDATED));`?